code/modules/reagents/chemistry/reagents/alcohol.dm

```diff
@@ -1776,3 +1788,14 @@
 	mutated = TRUE
 
 	return ..()
+
+/datum/reagent/consumable/ethanol/snakebite
+	name = "Snakebite"
+	id = "snakebite"
+	description = "A potent beer cocktail mixing lager and cider, and an alcoholic's favourite science experiment."
```

Comment:
I think we do use more "favorite" instead of "favourite", no idea which one we prefer so..up to you.